pkgs/tools/typesetting/tex/texlive/bin.nix

```diff
@@ -6,6 +6,7 @@
 , libpaper, graphite2, zziplib, harfbuzz, potrace, gmp, mpfr
 , brotli, cairo, pixman, xorg, clisp, biber, woff2, xxHash
 , makeWrapper, shortenPerlShebang
+, withCrossBootstrapTools ? false
```

Comment:
I suggest moving `himktables` to a separate output, rather than creating two copies of `bin.core`. The rest looks promising but I don't understand cross compilation!

Reply:
oh, I haven't thought about that. Yes, that's definitely better. Will update.